lib/tools/emacs/erlang.el

```diff
@@ -4,7 +4,7 @@
 ;; Author:   Anders Lindgren
 ;; Keywords: erlang, languages, processes
 ;; Date:     2011-12-11
-;; Version:  2.7.0
+;; Version:  2.8.0-cvs
```

Comment:
Means that this is a snapshot version, not a stable one. Newer Emacsen support as alternatives `-git` and `-snapshot`, but not 24.1.